src/lambda/commands/invokeLambda.ts

```diff
@@ -125,9 +125,15 @@ function createMessageReceivedFunc({
                     return undefined
                 }
 
+                let fileContent: string = ''
+                try {
+                    fileContent = readFileSync(fileLocations[0].fsPath, { encoding: 'utf8' })
+                } catch (e) {
+                    getLogger().error('readFileSync: Failed to read file at path %O', fileLocations[0].fsPath, e)
+                }
                 restParams.onPostMessage({
```

Comment:
Wrap this in the `try` statement. The way this is written, you'll post a message containing a blank string.